assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -263,4 +258,12 @@
 			justify-content: flex-start;
 		}
 	}
+
+	.googlesitekit-widget--ideaHubIdeas .googlesitekit-widget__footer {
+		padding: 14px 24px;
+	}
+
+	.googlesitekit-widget--ideaHubIdeas .googlesitekit-idea-hub__footer {
+		padding: 0;
+	}
```

Comment:
This reset not mentioned in IB.  Padding is being applied to this by a `.mdc-layout-grid` class